ui/homework/homework_1/test1.ts

```diff
@@ -4,6 +4,9 @@
 */
 
 type Person = {
+  name?: string;
+  age?: number | 'unknown';
```

Comment:
are you sure about this type?

Reply:
@aleksandrashmoylova  I understood, that in the TypeScript there is no type: 'unknown'. Logically, 'age': should be a 'number', but if we leave it that way, tslint will print a message on line 26: age: 'unknown'.

Do I need to leave the type of age 'number' and remove line 26 or do I only need to change the 'age' type?

```
**age?: number;**
... 
  {
    name: 'Batman',
    **// age: 'unknown',**
    isSuperHero: true,
    occupation: 'vigilante',
  }
...
```

Reply:
no, you don't need to disable the age in the object, type age can be different, try one more time please

Reply:
```
type Person = {
  ...
  age?: number | string;
  ...
};
```

I think it will be right. Anyway, I read that there is a type in TS called unknown, maybe I should use it.

Reply:
yes, but unknown and 'unknown' are different types